mono-traversable/test/Main.hs

```diff
@@ -149,6 +149,26 @@ main = hspec $ do
         prop "works on lists" $ \(Positive i) j ->
             ocompareLength (replicate i () :: [()]) j @?= compare i j
 
+    describe "groupBy" $ do
+        let test name dummy = prop name $ \(QCM.NonEmpty xs) -> do
```

Comment:
Why did you use NonEmpty?

Reply:
Copy and paste error. This is corrected.